### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by George-Nyamao**
I have done it!

### CODE REVIEW

File: nyamaog.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Jan  3 14:07:49 2020
+
+@author: gmnya
+"""
+
+def nyamaog(a:int) -> str:
```
Comment (kylestormcloud) [inline nyamaog.py:8]: Takes in an integer

**Review by ahtouw — APPROVED** [review]

**Comment by ahtouw** [comment]
nice!

File: nyamaog.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+-----------
+a should be a 1 or a 2
+"""
+    a
+    if a == 1:
+        return "This is my first string"
+    elif a == 2:
+        return "My second one"
+    else:
+        return "I think you're lost! Go back!"
```
Comment (kylestormcloud) [inline nyamaog.py:20]: Returns one of 3 strings

Comment (George-Nyamao) [reply nyamaog.py:20]: You're good to go!

**Review by kylestormcloud — APPROVED** [review]
You are approved!

**Pull request merged**